### PR TITLE
Dev/bindings

### PR DESCRIPTION
**Pull request by vcanogil**
Precursor to minimal viable product release. 
This can be released and used natively in Julia with the code parsing various data types from Python to Julia correctly.
The only outstanding problem is that user would need to install cclib themselves. Working on that currently.

The module can be used as follows: 
```
using Cclib 

data = ccread("file")
```

This will create a dictionary from which all the information can be accessed

### CODE REVIEW

**Comment by shivupa** [comment]
Lets update CI to do nightly + Julia stable (currently 1.9.1) + maybe LTS 1.6.7

1.2 and 1.7 are failing for reasons that we dont need to worry about. 

Nightly log could use some looking at.

**Comment by vcanogil** [comment]
> Lets update CI to do nightly + Julia stable (currently 1.9.1) + maybe LTS 1.6.7
> 
> 1.2 and 1.7 are failing for reasons that we dont need to worry about.
> 
> Nightly log could use some looking at.

This has been fixed. There's a mismatch in the working directory between prod and local for some reason.

**Review by berquist — APPROVED** [review]
Can you rebase against latest `main`?

File: src/config.jl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "aooverlaps" => Array{Float64, 2},
+    "atombasis" => Array{Array{Int64}},
+    "atomcharges" => Dict{String, Array{Float64}},
+    "atomcoords" => Array{Float64, 3},
+    "atommasses" => Array{Float64},
+    "atomnos" => Array{Int64},
+    "atomspins" => Dict{Array},
+    "ccenergies" => Array{Float64, 2},
+    "charge" => Int64,
+    "coreelectrons" => Array{}, # specify subytpe?
```
Comment (berquist) [inline src/config.jl:14]: These types are fine for now but we will definitely want to refine them as we go.

**Review by berquist — APPROVED** [review]
I haven't tried this interactively yet, but that's partially what CI is for :smiley:

+1 to Shiv's comment about updating Julia versions in CI.

**Comment by berquist** [comment]
> Nightly log could use some looking at.

Not sure what Shiv meant by this, but I did see https://github.com/cclib/Cclib.jl/actions/runs/5477200578/jobs/9975878249?pr=6#step:9:35.

**Comment by shivupa** [comment]
That was when CI was failing (because of a path issue victor fixed)

**Pull request merged**